integration/spark/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessBuilderOnSpark.scala

```diff
@@ -342,7 +347,12 @@ object DataLoadProcessBuilderOnSpark {
       .mapPartitionsWithIndex { case (index, rows) =>
         ThreadLocalSessionInfo.setConfigurationToCurrentThread(conf.value.value)
         DataLoadProcessorStepOnSpark
-          .convertFunc(rows, index, modelBroadcast, partialSuccessAccum, convertStepRowCounter)
+          .convertFunc(rows,
+            index,
+            modelBroadcast,
+            false,
```

Comment:
not required to pass after making isCompactionFlow as the last argument

Reply:
done